build/ci/e2e/Jenkinsfile

```diff
@@ -21,6 +21,16 @@ pipeline {
                 checkout scm
             }
         }
+        stage('Run Checks') {
```

Comment:
Does it really make sense to include check in e2e tests? For me it looks like it's enough to check in PR's only. It will be some sort of gatekeeping. After PR is merged all other jobs will work with code which was properly checked/formatted ...

Reply:
I suppose it's not strictly necessary unless someone ignores the PR check and merges to master anyway (or merges directly to master without a PR).

Reply:
I assume that people outside of our team can't do that. And if we will do something like that, then probably there will be a HUGE reason for that. 

I didn't mean that I'm against this check in other jobs. For me it just feels like it's enough to keep it only for PR job